internal/cmd/configuration/command_list.go

```diff
@@ -0,0 +1,50 @@
+package configuration
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List configuration fields in ~/.confluent/config.json.",
```

Comment:
list `configurable` config fields? make it clear it not all configs that'll be listed

Reply:
Yeah, I think it would be better to list out only the config fields that we're fine with users modifying, since we want to abstract away most of the file's contents from users.

Reply:
> I think it would be better to list out only the config fields that we're fine with users modifying via set

:+1:

Reply:
Yeah, they will still be able to see the value of `disable_updates`.

Reply:
Why would CP users not be able to update their binary? Also is there anywhere in CLI code that checks if a user is a CP user?

Reply:
CP is packaged for DEB/RPM, which does not allow self-updating. Here's where it's enforced (currently, not very well): https://github.com/confluentinc/cli/blob/10cee7c4ed6e72ce62ff1d44b77049796920f28b/packaging/confluent.sh#L60

Reply:
I'm not sure if we can actually know if a CLI binary is used for CP or not, since it looks like[ it's not stored alongside other CP tools](https://docs.confluent.io/platform/current/installation/cli-reference.html#confluent-cli-tool).

Reply:
True... we will in the future though, once we package the non-updating CLI with CP (instead of using the hacky workaround I described above)

Reply:
> True... we will in the future though, once we package the non-updating CLI with CP (instead of using the hacky workaround I described above)

Yeah sounds good, I'll probably just make a ticket for that.